js/webapp.js

```diff
@@ -499,6 +503,7 @@ function WebApp(options){
       maxAge: sessionTimeoutMs
     }
   }));
+  loopbackConfig = zluxUtil.getLoopbackFromConfig(options);
```

Comment:
How does this work? loopbackConfig is a const: https://github.com/zowe/zlux-proxy-server/pull/43/files#diff-55d2935735e0c392902cca19a95e92c0R480